cpp/scenario/gazebo/include/scenario/gazebo/utils.h

```diff
@@ -180,6 +181,44 @@ namespace scenario {
              *         converted, an empty string otherwise.
              */
             std::string URDFStringToSDFString(const std::string& urdfString);
+
+            /**
+             * Normalize a vector in [-1, 1].
+             *
+             * The input, low and high arguments are broadcasted to a common
+             * size. Refer to the following for broadcasting definition:
+             *
+             * https://numpy.org/doc/stable/user/basics.broadcasting.html
+             *
+             * @throws std::invalid_argument If the arguments cannot be
+             * broadcasted.
+             * @param input The input vector.
+             * @param low The lower limit.
+             * @param high The higher limit.
+             * @return The normalized input.
+             */
```

Comment:
To be honest, I had no idea what you meant by "normalization"  by looking at this description, probably it could make sense to  say that element wise the operation performed is: 
~~~
output = 2.0 * (input - low) / (high - low) - 1; 
~~~
Or  if there is any link that defines it, it would be worth to link it.

Reply:
Normalization, whitening, etc there are a lot of similar terms. I'll add the equation in the documentation.

Reply:
Documentation update in 3a3fb12